values.yaml

```diff
@@ -262,6 +262,7 @@ console:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  topologySpreadConstraints: ""
```

Comment:
Wouldn't it make more sense to use a list? Or is there some use case here where templating is particularly useful?

Reply:
Maybe, and I am certainly open to that.  Using a tpl function allows something like this:

```
  topologySpreadConstraints: |
    - maxSkew: 1
      topologyKey: "topology.kubernetes.io/zone"
      whenUnsatisfiable: DoNotSchedule
      labelSelector:
        matchLabels:
          {{- include "jitsu.selectorLabels" . | nindent 10 }}
          {{- include "jitsu.ingest.selectorLabels" . | nindent 10 }}
    - maxSkew: 1
      topologyKey: "kubernetes.io/hostname"
      whenUnsatisfiable: DoNotSchedule
      labelSelector:
        matchLabels:
          {{- include "jitsu.selectorLabels" . | nindent 10 }}
          {{- include "jitsu.ingest.selectorLabels" . | nindent 10 }}
```

where you don't have to know as much about the labels.

Reply:
I agree that's worth supporting. See my other comment for a suggestion on how to support both.